com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -187,7 +187,7 @@ else if (hostPlatform == Platform.X86_64Linux)
     private static Result exec(List<String> args) throws IOException {
         logger.info("exec: " + String.join(" ", args));
         Map<String, String> env = new HashMap<String, String>();
-        if (Platform.getHostPlatform() == Platform.X86_64Linux || Platform.getHostPlatform() == Platform.X86Linux) {
+        if (Platform.getHostPlatform() == Platform.X86_64Linux || Platform.getHostPlatform() == Platform.Arm64Linux) {
```

Comment:
We haven't fully decided if arm64-linux is a "desktop" platform, but I think we're leaning towards that.
Might be useful if someone develops/run on e.g. a Raspberry PI